### PR TITLE
Fp 3945

### PR DESCRIPTION
**Pull request by silversurfer5150**
#### What does this PR do? (please provide any background)

Adds a helper function to be called in the confirmation_body dust template

#### How can this be tested?
see main PR 

#### Any tech debt?
One extra function

#### What gif best describes how you feel about this work?
:)

---

### CODE REVIEW

File: helpers/isCarParking.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = function(dust) {
+
+ /*
+  * @description Detect whether the upgrade type is car parking for conditional rendering in confirmation template
+  * @param upgradeName is the name of the upgrade type which is used to check if it is car parking
+  * @example {@_checkIfCarParking upgradeName=content.name /} returns true if Car Parking otherwise false
```
Comment (kevinhodges) [inline helpers/isCarParking.js]: this example doesn't align with the exposed interface :)

File: helpers/isCarParking.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  */
+
+  dust.helpers._isCarParking = function(chunk, context, bodies, params) {
+    // strip the whitespace and lowercase content.name just to be safe and return true or false appropriately
+    return params.upgradeName.toLowerCase().replace(/ /g, "") === 'carparking' ? true : false;
```
Comment (putvande) [inline helpers/isCarParking.js]: Should there be a check for `param.upgradeName`? If no value has been passed this could break.

**Comment by silversurfer5150** [comment]
Updated this because I can see another park uses '1 Day Car Parking'
I removed the string equal to and instead went for indexOf()

File: helpers/isCarParking.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ /*
+  * @description Detect whether the upgrade type is car parking for conditional rendering in confirmation template
+  * @param upgradeName is the name of the upgrade type which is used to check if it is car parking
+  * @example {@_isCarParking upgradeName=content.name /} returns true if Car Parking otherwise false
+  */
+
+  dust.helpers._isCarParking = function(chunk, context, bodies, params) {
+    // strip the whitespace and lowercase content.name just to be safe and return true or false appropriately
+    if (typeof params !== 'undefined' && params.upgradeName) {
+      return (params.upgradeName.toLowerCase().replace(/ /g, '').indexOf('carparking')) !== -1;
```
Comment (putvande) [inline helpers/isCarParking.js]: Wonder if `/car\s*parking/i.test(params.upgradeName)` would be better

Comment (silversurfer5150) [reply helpers/isCarParking.js]: Yep, very slick i like it ;)

**Pull request merged**